StRoot/StJetMaker/emulator/StMuTrackEmu.h

```diff
@@ -42,6 +42,11 @@ class StMuTrackEmu {
     , _trackIndex(0)
     , _id(0)
     , _detectorId(0)
+    , _btofTrayId(0)        // bassam
+    , _nSigmaTofPion(0)     // bassam
+    , _nSigmaTofKaon(0)     // bassam
+    , _nSigmaTofProton(0)   // bassam
+    , _nSigmaTofElectron(0) // bassam
```

Comment:
Pointless comments again... If you want to give credit to someone you'd better mention them in the log message